care/facility/models/patient.py

```diff
@@ -0,0 +1,161 @@
+from django.db import models
+from fernet_fields import EncryptedCharField, EncryptedTextField
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'fernet_fields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5331745)